pages/constitution.vue

```diff
@@ -0,0 +1,1377 @@
+<template>
```

Comment:
Isn't the original document written in markdown? I've set up markdown support, so we might as well use it